KonyAccelerate/controllers/mobile/frmWifiInfoController/frmWifiInfoController.js

```diff
@@ -1,23 +1,78 @@
         			If the wifi info is already fetched it will display the 
                     result or else it will fetch the wifi info 
         * @private
     */
-    formPreShowAction: function() {
-      let accelerateEventData = kony.store.getItem("accelerateEventData");
-      if(!kony.sdk.isNullOrUndefined(accelerateEventData)){
-        let wifiInformation = accelerateEventData.wifiInformation;
-        let wifiUserName = (wifiInformation.hasOwnProperty("wifi_name")) ? wifiInformation.wifi_name : "";
-        let wifiPassword = (wifiInformation.hasOwnProperty("wifi_password")) ? wifiInformation.wifi_password : "";
-        this.view.lblNetworkName.text = wifiUserName;
-        this.view.lblPasswordValue.text = wifiPassword;
-      }
+    formPostShowAction: function() {
+        let accelerateEventData = kony.store.getItem("accelerateEventData");
+        if (!kony.sdk.isNullOrUndefined(accelerateEventData)) {
+            let wifiInformation = accelerateEventData.wifiInformation;
+            let wifiUserName = (wifiInformation.hasOwnProperty("wifi_name")) ? wifiInformation.wifi_name : "";
+            let wifiPassword = (wifiInformation.hasOwnProperty("wifi_password")) ? wifiInformation.wifi_password : "";
+            this.view.lblNetworkName.text = wifiUserName;
+            this.view.lblPasswordValue.text = wifiPassword;
+          	this.view.flxWifiImageContainer.isVisible = true;
+          	this.animateWifiCard("62%");
+        } else {
+            fetchObjectData(eventConstants.OBJECT_SERVICE_NAME, eventConstants.WIFI_OBJECT_NAME, {}, this.wifiFetchSuccess.bind(this), this.wifiFetchFailure.bind(this));
+        }
+      	
     },
+    /**
+     * @function wifiFetchSuccess
+     * @description This function is invoked in the success of wifi object and used to set the data to form
+     * @param successResponse The success response of the wifi object
+     * @private
+     */
+    wifiFetchSuccess: function(successResponse) {
+        let wifiInfo = successResponse.hasOwnProperty("records") ? successResponse.records : null;
+        if (wifiInfo !== null) {
+            let wifiData = wifiInfo[0];
+            let wifiUserName = (wifiData.hasOwnProperty("wifi_name")) ? wifiData.wifi_name : "";
+            let wifiPassword = (wifiData.hasOwnProperty("wifi_password")) ? wifiData.wifi_password : "";
+            this.view.lblNetworkName.text = wifiUserName;
+            this.view.lblPasswordValue.text = wifiPassword;
+          	this.view.flxWifiImageContainer.isVisible = true;
+          	this.animateWifiCard("62%");
+        } else {
+            kony.print("Unable to fetch Wifi Info");
+        }
+    },
+    /**
+     * @function wifiFetchFailure
+     * @description This function is invoked in the failure of wifi object and used to set the data to form
+     * @param failureResponse The failure response of the wifi object
+     * @private
+     */
+    wifiFetchFailure: function(failureResponse) {
```

Comment:
Can it be ```wifiServiceFail```?

Reply:
Actually we are using the same naming convention for object service.
"[ObjectName][Operation][Success/Failure]"